Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/Source.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
+{
+    /// <summary>
```

Comment:
Again, as general comment - please remove leading whitespace in comments.

Reply:
Removed tab-indentation as mentioned in other comments, but just to make sure, does this mean a single space between /// and text because those comes automatically by Visual Studio. (as shown in this comment's image.)